core/designsystem/build.gradle.kts

```diff
@@ -11,6 +11,7 @@ kotlin {
 }
 
 dependencies {
+    implementation(projects.core.model)
```

Comment:
I want to access String resources in the designsystem, so I added a dependency, is there any problem?